python/vpc-ec2-local-zones/user_data/db_mysql

```diff
@@ -0,0 +1,26 @@
+#cloud-config
+packages:
+ - docker
+write_files:
+- content: |
+    version: '3.1'
+    services:
+      db:
+        image: mysql:5.7
```

Comment:
This version will soon be out of support. Is it possible to use the latest version?

Reply:
thanks. I removed any version specification to use the latest mysql image